lib/Doctrine/DBAL/Schema/Schema.php

```diff
@@ -283,11 +248,11 @@ public function getSequences()
      *
      * @param string $namespaceName The name of the namespace to create.
      *
-     * @return \Doctrine\DBAL\Schema\Schema This schema instance.
+     * @return $this
```

Comment:
Please avoid `$this`, use `static` or nothing instead.

Reply:
I'd rather leave it as `$this`. Please see http://docs.phpdoc.org/guides/types.html#keywords.